utils/docker/configure-tests.sh

```diff
@@ -74,6 +74,17 @@ TEST_BUILD="debug nondebug"
 TEST_PROVIDERS=sockets
 EOF
 
+#configure python tests
+	cat << EOF >> $WORKDIR/src/test/testconfig.py
+config = {
+	'unittest_log_level': 1,
+	'pmem_fs_dir': '/mnt/pmem',
```

Comment:
this one must also be `/tmp` on travis. Also set PMEM_FS_DIR_FORCE_PMEM=1 if it exists.